pipelines/generate_predictive.py

```diff
@@ -53,7 +53,7 @@ def generate_and_save_predictions(
         description=("Do posterior prediction from a pyrenew-hew fit.")
     )
     parser.add_argument(
-        "model-run-dir",
+        "--model_run_dir",
```

Comment:
```suggestion
        "model_run_dir",
```

I think this would be my preference. Maybe we could sneak in a change throughout the repo:

- [Instances of `"model-run-dir"`](https://github.com/search?q=repo%3ACDCgov%2Fpyrenew-hew%20model-run-dir&type=code)
- [Instances of `"--model_run_dir"`](https://github.com/search?q=repo%3ACDCgov%2Fpyrenew-hew+--model_run_dir&type=code)

Reply:
And let's merge this into main instead of [`add-demo-data`](https://github.com/CDCgov/pyrenew-hew/tree/add-demo-data)

Reply:
@dylanhmorris suggests this convention:
`filename.py positional_argument --named-argument namedvalue --other-named-argument othernamedvalue`

Which happens to line up with what I suggested, though I did not know that at the time.

Reply:
> @dylanhmorris suggests this convention: `filename.py positional_argument --named-argument namedvalue --other-named-argument othernamedvalue`
> 
> Which happens to line up with what I suggested, though I did not know that at the time.

This is in a bit of conflict with two bits of the code base atm:
- `iteration_helper` shell scripts are expecting kwargs
- `argparse` in the python scripts is expecting to pass through `kwargs` only.

The second conflict is not detected by `forecast_state.py` because the underlying script functions are call directly.

Reply:
I'm not sure, but @SamuelBrand1 has made changes to the iteration_helper scripts in https://github.com/CDCgov/pyrenew-hew/pull/138, so maybe that resolves it?